### PR TITLE
Bound to other IP-Addresses

### PR DESCRIPTION
**Pull request by irinil**
Bound to different ip-address in the same network not only localhost.

### CODE REVIEW

File: coap-core/src/main/java/com/mbed/coap/server/CoapServer.java [hunk 1]
```diff
@@ -39,6 +39,7 @@
 import com.mbed.coap.utils.RequestCallback;
 import java.io.IOException;
 import java.net.InetSocketAddress;
+import java.util.ArrayList;
```
Comment (szysas) [inline coap-core/src/main/java/com/mbed/coap/server/CoapServer.java:42]: Remove this.

File: coap-core/src/main/java/com/mbed/coap/server/CoapServerBuilder.java [hunk 2]
```diff
@@ -143,6 +144,11 @@ public CoapServerBuilderForUdp transport(int port) {
             transport(new DatagramSocketTransport(new InetSocketAddress(port)));
             return this;
         }
+        
+        public CoapServerBuilderForUdp transport(InetAddress address,int port) {
```
Comment (szysas) [inline coap-core/src/main/java/com/mbed/coap/server/CoapServerBuilder.java:148]: Can you add test?

**Review by szysas — CHANGES_REQUESTED** [review]

File: coap-core/src/main/java/com/mbed/coap/server/CoapServer.java (hunk 1 above)
Comment (irinil) [reply coap-core/src/main/java/com/mbed/coap/server/CoapServer.java:42]: Ok :)

File: coap-core/src/main/java/com/mbed/coap/server/CoapServerBuilder.java (hunk 2 above)
Comment (irinil) [reply coap-core/src/main/java/com/mbed/coap/server/CoapServerBuilder.java:148]: Yes of course!

**Pull request merged**